src/app/feature-flag/notification-footer/feature-footer.component.less

```diff
@@ -0,0 +1,73 @@
+@import (reference) '../../../assets/stylesheets/shared/osio.less';
+
+.feature-footer {
+  position:absolute;
+  bottom:0;
+  width:100%;
+  height:60px;   /* Height of the footer */
```

Comment:
This comment is not necessary. The class `.feature-footer` describes what this height is for.